blog/2020-05-13-native-server-announce.adoc

```diff
@@ -0,0 +1,91 @@
+# Generating Identities yaml using USER and PASS env vars.                     #
+################################################################################
+2020-05-13 09:35:09,753 INFO  [io.quarkus] (main) config-generator 2.0.0.Dev01 (powered by Quarkus 1.4.1.Final) started in 0.075s. 
+2020-05-13 09:35:09,759 INFO  [io.quarkus] (main) Profile prod activated. 
+2020-05-13 09:35:09,759 INFO  [io.quarkus] (main) Installed features: [cdi, qute]
+2020-05-13 09:35:09,789 INFO  [io.quarkus] (main) config-generator stopped in 0.000s
+__  ____  __  _____   ___  __ ____  ______ 
+ --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
+ -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
+--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
+2020-05-13 09:35:10,394 INFO  [ListenerBean] (main) The application is starting...
+2020-05-13 09:35:10,571 INFO  [org.inf.CONTAINER] (main) ISPN000128: Infinispan version: Infinispan 'N/A' 11.0.0.Dev05
+2020-05-13 09:35:10,578 INFO  [org.inf.CLUSTER] (main) ISPN000078: Starting JGroups channel infinispan
+2020-05-13 09:35:15,655 INFO  [org.inf.CLUSTER] (main) ISPN000094: Received new cluster view for channel infinispan: [localhost-38883|0] (1) [localhost-38883]
+2020-05-13 09:35:15,656 INFO  [org.inf.CLUSTER] (main) ISPN000079: Channel infinispan local address is localhost-38883, physical addresses are [127.0.0.1:7800]
+2020-05-13 09:35:15,660 INFO  [org.inf.CONTAINER] (main) ISPN000390: Persisted state, version=11.0.0.Dev05 timestamp=2020-05-13T09:35:15.657080Z
+2020-05-13 09:35:15,682 INFO  [org.inf.CONTAINER] (main) ISPN000104: Using EmbeddedTransactionManager
+2020-05-13 09:35:15,738 INFO  [org.inf.SERVER] (ForkJoinPool.commonPool-worker-3) ISPN080018: Protocol HotRod (internal)
+2020-05-13 09:35:15,741 INFO  [org.inf.SERVER] (main) ISPN080018: Protocol REST (internal)
+2020-05-13 09:35:15,754 INFO  [org.inf.SERVER] (main) ISPN080004: Protocol SINGLE_PORT listening on 127.0.0.1:11222
+2020-05-13 09:35:15,754 INFO  [org.inf.SERVER] (main) ISPN080034: Server 'localhost-38883' listening on http://127.0.0.1:11222
+2020-05-13 09:35:15,754 INFO  [org.inf.SERVER] (main) ISPN080001: Infinispan Server 11.0.0.Dev05 started in 5490ms
+2020-05-13 09:35:15,754 INFO  [io.quarkus] (main) infinispan-quarkus-server-runner 11.0.0.Dev05-01 (powered by Quarkus 1.3.1.Final) started in 5.649s. 
+2020-05-13 09:35:15,755 INFO  [io.quarkus] (main) Profile prod activated. 
+2020-05-13 09:35:15,755 INFO  [io.quarkus] (main) Installed features: [cdi, infinispan-embedded, infinispan-server]
+----
+
+== Further Reading
+For more detailed information abou how to use the `infinispan/server` and `infinispan/server-native` image, please consult the
+https://github.com/infinispan/infinispan-images/blob/master/README.md[official documentation].
+
+== Get it, Use it, Ask us!
+The Quarkus extension and the `server-native` image are currently in a Alpha state, so please try it out and let us know
+if you run into any issues.
```

Comment:
We probably should mention there will be quite a few bugs until we can iron it out.

Reply:
That's kinda what I meant with "Alpha", but I'll make that more clear :+1: